predict-taxi-trip-duration/NYC Taxi 3 - Simple Featuretools.ipynb

```diff
@@ -247,9 +251,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<p>We can then use the `entity_from_dataframe` method to add an Entity called <i>trips</i>. We want to track the `id`, the `time_index` and specify other types of variables we care about in this entity. </p>\n",
+    "We can then use the `add_dataframe` method to add a dataframe called <i>trips</i>. We want to track the `id`, the `time_index` and specify other column types we care about in this table.\n",
     "\n",
-    "<p>As a note: Featuretools will try to interpret the types of variables. We can override this interpretation by specifying the types. In this case, I wanted <b>passenger_count</b> to be a type of Ordinal, and <b>vendor_id</b> to be of type Categorical.</p>"
+    "<p>As a note: Featuretools will try to interpret the types of columns. We can override this interpretation by specifying the types. In this case, I wanted <b>passenger_count</b> to be a type of Ordinal, and <b>vendor_id</b> to be of type Categorical.</p>"
```

Comment:
Worth mentioning Woodwork here and linking to the logical types guide? https://woodwork.alteryx.com/en/stable/guides/logical_types_and_semantic_tags.html

Reply:
Yeah, added a sentence with that link.